client/layout/logged-out-oauth.jsx

```diff
@@ -3,17 +3,15 @@
  */
 import React from 'react';
 
-module.exports = React.createClass( {
-	displayName: 'LayoutLoggedOutAuth',
-
-	render: function() {
-		return (
-			<div className="wp logged-out-auth">
-				<div id="content" className="wp-content">
-					<div id="primary" className="wp-primary wp-section">
-					</div>
-				</div>
+const LayoutLoggedOutAuth = () => (
+	<div className="wp logged-out-auth">
```

Comment:
This turns out to be a bit of a rabbit hole -- I can rename them, but then again, the class `logged-out-auth` is styled in `client/auth/style.scss`; and in that directory, there are lines like `import AuthStore from 'lib/oauth-store';`. So apparently this is pretty pervasive. 

I'll leave as is, since I guess there's no point in changing only one occurrence, and changing them all is clearly out of scope.